builder/builder.go

```diff
@@ -1151,6 +1151,46 @@ func (b *Builder) EditBundles(bundles []string, suppressEditor bool, add bool, g
 	return nil
 }
 
+// ValidateBundles runs bundle parsing validation on a list of local bundles, or
```

Comment:
Optional: the ValidateBundles don't actually call our bundle validation function. These are different validations. Maybe rename one of them to avoid confusion. Context: I was initially puzzled how you could validate just the local bundles.

Reply:
Ah, I see why you were thrown by that name. 

I had considered other names. I don't think the using "validate" in both is bad; both are validating things, they're just validating different things -- one is "validating a `bundleSet`, the other is "validating a list of bundle files". To make the name more descriptive, I considered calling this function `ValidateBundleFiles`, but 1) I wanted it to keep parallel with the other `<Action>Bundles` functions (Add, Remove, Edit), and 2) that name makes me think it should be taking filepaths. And maybe it should? But again a) I wanted to keep it parallel with the others that all take a list of bundle names, and b) that would push the path lookup to the caller (the `bundleValidateCmd.Run`), and i) that would require exporting `getBundlePath`, and ii) I favor keeping the CLI layer thin and decoupled from how bundle paths are dealt with.

Man, I had to bust out Roman Numerals to keep those lists separate. Clear sign I wrote too complicated of a response! :stuck_out_tongue_winking_eye:

Reply:
Is the name 'check' a way to go for evaluation the bundles in isolation?